go.mod

```
@@ -5,28 +5,17 @@ require (
 	github.com/aelsabbahy/GOnetstat v0.0.0-20160428114218-edf89f784e08
 	github.com/aelsabbahy/go-ps v0.0.0-20170721000941-443386855ca1
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/cheekybits/genny v0.0.0-20160824153601-e8e29e67948b
-	github.com/docker/docker v0.0.0-20161109014415-383a2f046b16
-	github.com/fatih/color v0.0.0-20161025120501-bf82308e8c85
-	github.com/fsnotify/fsnotify v1.4.7 // indirect
-	github.com/golang/protobuf v1.3.2 // indirect
-	github.com/hpcloud/tail v1.0.0 // indirect
-	github.com/kr/pretty v0.1.0 // indirect
-	github.com/mattn/go-colorable v0.0.9 // indirect
-	github.com/mattn/go-isatty v0.0.4 // indirect
-	github.com/miekg/dns v0.0.0-20161018060808-58f52c57ce9d
+	github.com/cheekybits/genny v1.0.0
+	github.com/docker/docker v1.13.1
+	github.com/fatih/color v1.9.0
+	github.com/miekg/dns v1.1.29
 	github.com/oleiade/reflections v0.0.0-20160817071559-0e86b3c98b2f
-	github.com/onsi/ginkgo v1.10.2 // indirect
-	github.com/onsi/gomega v0.0.0-20161031154339-ff4bc6b6f9f5
+	github.com/onsi/gomega v1.9.0
 	github.com/opencontainers/runc v0.0.0-20161107232042-8779fa57eb4a
-	github.com/patrickmn/go-cache v2.0.0+incompatible
+	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/stretchr/testify v1.4.0
 	github.com/urfave/cli v0.0.0-20161102131801-d86a009f5e13
-	golang.org/x/sys v0.0.0-20181210030007-2a47403f2ae5 // indirect
-	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
-	gopkg.in/fsnotify.v1 v1.4.7 // indirect
-	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
-	gopkg.in/yaml.v2 v2.2.2
+	gopkg.in/yaml.v2 v2.2.8
 )
 
 go 1.13
```

```suggestion
go 1.14
```

If we're not verifying against 1.13 we probably should update this to 1.14 to be consistent.